### PR TITLE
KenrickHTML

### PR DESCRIPTION
**Pull request by Kenrixkk22**
adds HTML and Style sheet

### CODE REVIEW

File: lesson_00/kenricksutherland/README.md
Comment (anthonydmays) [inline lesson_00/kenricksutherland/README.md:1]: Why did you make a completely new README? Delete this and I'll use your original README to judge your HTML version.

File: lesson_01/Kenrick/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<p>In my daily routine, I prioritize a balanced blend of physical and mental activities to nurture both my body and mind. One of my cornerstone hobbies is working out, a commitment I uphold diligently four to five days a week. Whether it's weightlifting, cardio sessions I'm an avid reader, particularly drawn to self-development books like "Atomic Habits," which inspire me to refine my habits and maximize my potential. When I crave scenery of nature and need another perspective I lace up my running shoes and hit the track allowing each stride to carry me closer to both physical fitness and mental clarity. These hobbies not only are ones I enjoy but also serve as pillars of personal growth and fulfillment in my life.</p>
+
+<h2>Goals Of Mine ⭐</h2>
+<div class="Line"></div>
+<p>Firstly, on the fitness front, I aim to enhance my overall strength and endurance by consistently hitting the gym four to five times a week, focusing on progressive overload. I'm committed to obtaining a more balanced lifestyle by prioritizing mindfulness and self-care practices.</p>
+
+<h2>Favorite FOODS 🍕</h2>
+<div class="Line"></div>
+<ul>
+  <li>Chicken and Broccoli Alfredo</li>
```
Comment (anthonydmays) [inline lesson_01/Kenrick/index.html:26]: The space between the bullets and text is longer than it should be, please fix.

Comment (anthonydmays) [reply lesson_01/Kenrick/index.html:26]: Also, the alignment is weird. Should be flush left.

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Some significant differences between the html and markdown version. Continue making adjustments.

File: lesson_01/Kenrick/index.html [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
-  <li>Chicken Tenders</li>
-  <li>Steak</li>
-  <li>Mac & Cheese</li>
-  <li>Brownies</li>
-  <li>Tiramisu</li>
-</ul>
-
-<h2>Travel Experiences ✈️</h2>
-<div class="Line"></div>
+<h3>Favorite FOODS 🍕</h3>
```
Comment (anthonydmays) [inline lesson_01/Kenrick/index.html:28]: Bulleted list missing bullets

File: lesson_01/Kenrick/index.html [hunk 3]
```diff
@@ -1,60 +1,61 @@
...
+Mac & Cheese<br>
+Brownies<br>
+Tiramisu</p>
+
+<h3>Travel Experiences ✈️</h3>
+
 <p>I've been to Aruba, Curacao, and the Dominican Republic, enjoying their awesome beaches and cool vibes. Now, I'm itching to discover Jamaica for some laid-back vibes, Greece for Greek Mythology and beautiful beaches, Bermuda for those pink sands I keep hearing about, and Punta Cana for more of that tropical paradise feeling. Can't wait to explore more awesome places.</p>
 
-<img src="https://www.siliconrepublic.com/wp-content/uploads/2022/11/AdobeStock_20136083-718x523.jpeg" alt="Kenrick Sutherland">
+<h3>Favorite Quotes</h3>
```
Comment (anthonydmays) [inline lesson_01/Kenrick/index.html:45]: Bulleted list missing bullets.

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Almost there! Keep it going.

**Review by HeirJordan3 — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]
Missing airliner picture and list bullets.

**Pull request merged**